src/main/resources/application.yml

```diff
@@ -12,7 +12,7 @@ spring:
     username: postgres
     password: password
   mail:
-    host: mailhog.qa.argo.cancercollaboratory.org
```

Comment:
Ideally we would replace this with a generic URL instead of hardcoding a specific test environment.